src/main/java/org/folio/dew/service/JobCommandsReceiverService.java

```diff
@@ -104,6 +104,7 @@ public void postConstruct() {
   }
 
   @KafkaListener(
+    concurrency = "30",
```

Comment:
This 30 must be configurable using environment variable. And 30 should be by default. It is essential because we could face problems in prod and would need to decrease or increase the concurrency.

Reply:
@obozhko-folio

Reply:
done